reconcile/openshift_saas_deploy.py

```diff
@@ -30,6 +59,35 @@ def run(dry_run, thread_pool_size=10, io_dir='throughput/',
         logging.error('no saas files found')
         sys.exit(ExitCodes.ERROR)
 
+    # notify different outputs (publish results, slack notifications)
+    # we only do this if:
+    # - this is not a dry run
+    # - there is a single saas file deployed
```

Comment:
why is this?

Reply:
this integration can run either for all saas files or for a single one. each saas file has a different slack section, so we can't initialize one for all. so we only use this when a single saas file is deployed.